src/entities/note.cpp

```diff
@@ -2277,6 +2277,9 @@ QString Note::textToMarkdownHtml(QString str, const QString &notesPath,
        src=\")((?!file:\/\/).+)\")"),
                 "\\1file://" + windowsSlash + notePath + "/\\2\"");
     */
+    // css classes applicable for export and preview
+    QString additionalClasses = ".task-list-item-checkbox {list-style: none; margin-left: -14px; text-decoration: none;} "
```

Comment:
`additionalClasses` might not be the best name here. How about `listClasses / listCSS` or something along those lines.